crc-admin-helper.spec.in

```diff
@@ -78,6 +79,9 @@ go test ./...
 * Tue May 17 2022 Christophe Fergeau <cfergeau@redhat.com> - 0.0.11-1
 - Update to admin-helper 0.0.11
 
+* Tue May 31 2022 Anjan Nath <kaludios@gmail.com>
+- install macOS arm64 builds
+
```

Comment:
This would look like this
```
* Tue May 31 2022 Anjan Nath <kaludios@gmail.com> - 0.0.11-1
- Update to admin-helper 0.0.11
- Install macOS arm64 builds
```

Reply:
updated